sample/main.cpp

```diff
@@ -5,16 +5,15 @@
 
 class custom_type
 {
+    const std::string str_ { "user defined" };
+    const double pi_ { 3.14 };
+
```

Comment:
I prefer private members placed at the bottom of the class, but in this sample code its ok, just also update readme accordingly

Reply:
Fixed.